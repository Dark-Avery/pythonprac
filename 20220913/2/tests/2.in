[-256, 1024, 0.23, 213, 3.1415]