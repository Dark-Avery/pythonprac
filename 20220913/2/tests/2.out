-256, 0.23, 3.1415, 213, 1024
